tests/foreman/api/test_syncplan.py

```diff
@@ -178,7 +178,7 @@ def test_positive_create_with_sync_date(self):
                     sync_date=syncdate,
                 ).create()
                 self.assertEqual(
-                    syncdate.strftime('%Y-%m-%dT%H:%M:%SZ'),
+                    syncdate.strftime('%Y-%m-%d %H:%M:%S UTC'),
```

Comment:
Sure, I was not insisted on that, I just wanted to make sure that we will not miss anything